Harvest.Web/ClientApp/src/Tickets/TicketAttachments.tsx

```diff
@@ -65,8 +65,8 @@ export const TicketAttachments = (props: Props) => {
         ))}
       </ul>
       <FormGroup>
-        <Label>Attach files?</Label>
-        {ticket.completed !== false ? (
+        {!ticket.completed && <Label>Attach files?</Label>}
+        {!ticket.completed && (
           <FileUpload
```

Comment:
There is no reason to put these in two different lines -- they are the same conditional.  Just stick the attach files label inside and it should be fine.  might have to add an empty <> or a div but otherwise it should be fine.

Reply:
I keep forgetting about the empty <>